### ISSUE TITLE
feature request: jump to  preview automatically acording the markdown code

### ISSUE THREAD

**Issue by fastfading** [marp-team/marp-vscode#114]
sorry to bother you again , bro.  But I really like your work.
I just compare this to vscode  markdown preview enhanced extension
that markdown preview has a feature, when you click on the markdown code on the left, the preview will jump to that page in preview automatically.

can this extension do the same ?

**Comment by yhatt** [comment]
[Marp team](https://github.com/marp-team) is focusing to the creation of slide deck, and not have a lot of interest in the other features now. It would be better as an editor extension rather than implement as the feature for Marp extension.

I guess there is an extra [Markdown extension](https://code.visualstudio.com/docs/languages/markdown#_markdown-extensions) for to sync position in VS Code's native preview of Markdown. (but I could not found yet)

**Comment by synapticvoid** [comment]
Hi,
This behavior also drove me crazy while using the VSCode plugin, but as @yhatt said, it's not really the job of Marp to synchronize the panes scroll.
I ended up using this plugin : [Scroll to Cursor](https://marketplace.visualstudio.com/items?itemName=tobark.scroll-to-cursor). I've configured it to keep 3 visible lines at the top and with a simple `Ctrl + L`, the preview pane is perfectly centered.

It's not a "built-in" solution, but it works pretty well.
